sdk/core/core-client/review/core-client.api.md

```diff
@@ -18,6 +18,12 @@ import { PipelineResponse } from '@azure/core-rest-pipeline';
 import { TokenCredential } from '@azure/core-auth';
 import { TransferProgressEvent } from '@azure/core-rest-pipeline';
 
+// @public
+export interface AdditionalPolicyConfig {
```

Comment:
[NIT]
Changelog?

Reply:
I'll push a changelog update in a moment 👍